cmd/flipt/main.go

```diff
@@ -88,15 +89,35 @@ var (
 )
 
 func main() {
+	var (
+		once         sync.Once
+		loggerConfig = zap.Config{
+			Level:            zap.NewAtomicLevelAt(zap.InfoLevel),
+			Development:      true,
+			Encoding:         "console",
+			EncoderConfig:    zap.NewDevelopmentEncoderConfig(),
```

Comment:
What do you think about adding `DisableCaller: true,` here? This prevents the filename/line number from being added to the log output, which I think is only helpful when someone is actually working on Flipt itself, not running it

Reply:
💯 Agreed. I actually have that applied locally. Brings it closed to the original output.